azurerm/resource_arm_subnet.go

```diff
@@ -195,6 +201,22 @@ func resourceArmSubnetCreateUpdate(d *schema.ResourceData, meta interface{}) err
 		properties.RouteTable = nil
 	}
 
+	if v, ok := d.GetOk("disable_private_endpoint_network_policies"); ok {
```

Comment:
If this is just disabling network policies should we include `private_endpoint` in the name? Or does this only work if private endpoints are in use?

Reply:
same comment

Reply:
This is only for `private_link_endpoints`, I have changed the logic a bit so it only passes the value to the RP if it is set to `true`, which is the only time it makes sense to include.